index.js

```diff
@@ -0,0 +1,221 @@
+const openHamburger = document.getElementById('openHamburger');
+const closeHamburger = document.getElementById('closeHamburger');
+const navcontainer = document.querySelector('.bottom-nav_center');
+const navLink = document.querySelectorAll('.nav-list_item');
+
+openHamburger.addEventListener('click', () => {
+  navcontainer.classList.toggle('hidden');
+  openHamburger.classList.add('hidden');
+  closeHamburger.classList.remove('hidden');
+});
+
+closeHamburger.addEventListener('click', () => {
+  navcontainer.classList.toggle('hidden');
+  openHamburger.classList.remove('hidden');
+  closeHamburger.classList.add('hidden');
+});
+
+navLink.forEach((link) => link.addEventListener('click', () => {
+  navcontainer.classList.add('hidden');
+  openHamburger.classList.remove('hidden');
+  closeHamburger.classList.add('hidden');
+}));
+
+//= == End of Mobile Menu ===
+
+const SpecialChefs = [
+  {
+    name: 'Mr.William',
+    position: 'Executive Chef',
+    image: './images/chef-william.jpg',
+    about: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Architecto praesentium commodi beatae',
+  },
+  {
+    name: 'Mr.Adam',
+    position: 'Head Chef',
+    image: './images/chef-adam.jpg',
+    about: 'Lorem, ipsum dolor sit amet consectetur adipisicing elit. Architecto praesentium commodi beatae',
+  },
```

Comment:
- [ ]  While you make a great effort😊 I see that you forgot to use the article tags for this section, so it will be better if you use them as this from the requirements, so for this section💯😃, It contains a bottom section with a column that includes a section header, a divider, and six individual articles, each with an image, title, subtitle, and description. Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description..🚀👌

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/dd161398-daeb-465c-9c1c-61a4d172aebf)   |   ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/19a16f7e-4896-4dff-a47e-53464554fa97)   |